Note that `thirdPartyErrorFilterIntegration` is not supported with Loader or CDN bundles (#11053)
getsentry/sentry-docs: docs/platforms/javascript/common/configuration/filtering.mdx

If you choose a mode to only apply tags, the tags can then be used in Sentry to filter your issue stream with the `third_party_code` tag in the issue search bar.

<Note>

The `thirdPartyErrorFilterIntegration` will not work with the [Sentry Loader Script or CDN Bundles](https://docs.sentry.io/platforms/javascript/install/loader/).

> This is the case because the Sentry Loader Script and CDN Bundles would be detected by the integration as "third party", causing it to apply the chosen behaviour for almost all events, because the Sentry SDK wraps a lot of browser-native API.

Reviewer: L: This sentence is quite long. A suggestion:
```suggestion
This is the case because the Sentry Loader Script and CDN Bundles are detected as "third party" by the integration. This makes it apply the chosen behavior to almost all events, since the Sentry SDK wraps many browser-native APIs.
```
